### ISSUE TITLE
Add description and recommendation about unicode LTR and RTL marks

### ISSUE THREAD

**Issue by andoks** [mdn/content#45248]
### Proposal

Add description and recommendation about unicode LEFT-TO-RIGHT MARK ([U+200E](https://unicode-explorer.com/c/200E)) and RIGHT-TO-LEFT MARK ([U+200F](https://unicode-explorer.com/c/200F)) to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo.

- when to use and not to use
- pros and cons vs \<bdo> and \<bdi>
- accessibility concerns?
- 

### Browser support

Firefox

### Tasks

- [ ]

### Dependencies

- [ ]

### Are you willing to support this work?

I can provide reviews from the perspective of a learner - but my motivation for the topic is that I myself am not a subject matter expert, and maybe like to read about this subject at a source I trust, like MDN is.

**Comment by pepelsbey** [comment]
You’re right! Unfortunately, there is currently no RTL/LTR guide that combines all methods for working with different writing directions. Such a guide would be a perfect place to mention Unicode symbols and accessibility considerations.

**Comment by pepelsbey** [comment]
Moved it to content for the community to pick it. Unfortunately, we don’t currently have an RTL guide that it could be a good part of.

I don’t think adding a note to any bdo/bdi reference would suffice.

**Comment by Josh-Cena** [comment]
btw, there's recently significant expansion to https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/dir covering the base direction. We can probably add a guide about writing direction, but I'm extremely unsure if it should be HTML, CSS, or somewhere else. @pepelsbey You have any suggestions?
